model/src/main/proto/format_import_proto_library.bzl

```diff
@@ -0,0 +1,39 @@
+
+def format_import_proto_library(name, src, deps):
+  """
+  Formats the import statement of the src file to contain a full path to the file
+  The genrule() rule generates a copy of the src file and alters the copy
+  The proto_library() rule takes this altered copy and builds it
+  """
+
+  native.genrule(
+    name = name,
+    srcs = [src],
+    outs = ["processed_" + src],
+    cmd = '''
+    cat $< |
+    sed 's/import \"/import \"model\/src\/main\/proto\//g' |
+    sed 's/\"model\/src\/main\/proto\/exploration/\"model\/processed_src\/main\/proto\/exploration/g' |
+    sed 's/\"model\/src\/main\/proto\/topic/\"model\/processed_src\/main\/proto\/topic/g' |
+    sed 's/\"model\/src\/main\/proto\/question/\"model\/processed_src\/main\/proto\/question/g' > $@
+    '''
+    ,
+  )
+
+  native.proto_library(
+      name = name + "_proto",
+      srcs = ["processed_" + src],
+      deps = deps
+  )
+
+'''
+  native.java_lite_proto_library(
+      name = name + "_java_proto",
+      deps = [name + "_proto"] + java_lite_proto_deps,
+  )
+'''
+
+''' cp $< $@ &&\
```

Comment:
Is this & the above used? How?

Reply:
It is not used, I left it as insurance in the event that this implementation broke the builds in PRs depending on this change. I meant to remove it once I tested the other modules! I will remove it now :)